packages/website/blog/2024-05-27-announcing-typescript-eslint-v8-beta.mdx

```diff
@@ -215,10 +212,10 @@ To migrate to the new rules:
 - If you were disabling the ban on `{}`, consider enabling [`@typescript-eslint/no-empty-object-type`](https://v8--typescript-eslint.netlify.app/rules/no-empty-object-type), as it allows some cases of `{}` that were previously banned.
```

Comment:
I think most of these can all be updated to use the links on main now rather than the v8 links, if we want

Reply:
Ah, all except `no-restricted-types`. We never got around to backporting it. For consistency I'll stick with all of them in `v8`.